libs/sdk-ui-charts/src/charts/repeater/Repeater.tsx

```diff
@@ -56,7 +58,17 @@ export interface IRepeaterBucketProps {
 /**
  * @beta
  */
-export interface IRepeaterProps extends IBucketChartProps, IRepeaterBucketProps {}
+export interface IRepeaterProps extends IBucketChartProps, IRepeaterBucketProps {
+    /**
+     * Configure drillability; e.g. which parts of the visualization can be interacted with.
```

Comment:
Should we mention current limitations? (only attributes in columns are supported now). Maybe at least in the documentation? + We should update docs  - add drill props.